lib/matplotlib/colorbar.py

```diff
@@ -521,7 +522,7 @@ def _add_solids(self, X, Y, C):
         self.ax._hold = True
         col = self.ax.pcolormesh(*args, **kw)
         self.ax._hold = _hold
-        #self.add_observer(col) # We should observe, not be observed...
+        # self.add_observer(col) # We should observe, not be observed...
```

Comment:
If you are going to reformat commented out code I would just delete it (comment applies throughout).